.gitignore

```diff
@@ -9,3 +9,5 @@
 !/etc/build/.gitkeep
 
 /tests/Application/yarn.lock
+
+.env.test
```

Comment:
`/.env.test` is a good practice, as right now it would ignore `.env.test` files even in subdirectories (which is something not really expected here).